tests/test_transmissionCurve.py

```diff
@@ -0,0 +1,554 @@
+#
+# LSST Data Management System
+# Copyright 2017 LSST/AURA.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <http://www.lsstcorp.org/LegalNotices/>.
+#
+from __future__ import absolute_import, division, print_function
+
+import unittest
+
+import numpy as np
+
+import lsst.utils.tests
+import lsst.pex.exceptions
+import lsst.afw.geom
+import lsst.afw.image
+
+try:
+    type(display)
+except NameError:
+    display = False
+
+
+def makeTestCurve(random, x0, x1, y0=0.0, y1=0.0):
+    """Return a piecewise NumPy-friendly callable appropriate for testing
+    TransmissionCurve objects.
```

Comment:
The argument can be either a floating point number or a numpy-array thereof, since it only uses operators that numpy has overloaded.  Will document better.